srv/web/src/components/startingpages/StartingPage.tsx

```diff
@@ -17,18 +19,83 @@ export function Github() {
 window.location.href ='https://github.com/chronark/swapchain'
 }
 
-export default () => {
+/**
+ * This is a subset of all attributes from our mongo database.
+ *
+ * @interface Props
+ */
+export interface  Props { 
+   address?: string, 
+   timelock?: number, 
+   txFees?: string, 
+   bidType?: string, 
+   askType?: string, 
+   bidAmount?: number, 
+   expiryDate?: string,
+   askAmount?: number
+}
+
+/**
+ * Create the handler that sends user accs input to our database.
+ *
+ * @private
+ * @returns A post function to our database.
+ */
+export default class Form extends Component<Props> { 
```

Comment:
Please use functional components. Class components will be deprecated at some point.